src/types/boxed.rs

```diff
@@ -159,6 +160,8 @@ impl<T: Send + 'static> Clone for JsBox<T> {
     }
 }
 
+impl<T: Send + 'static> Object for JsBox<T> {}
```

Comment:
What is this about? Does this mean you can get and set properties on a `JsBox`? Does that work?

Reply:
Yes! It was an omission from a prior PR. `napi_external` (which `JsBox` wrap) inherit from `Object`.

I added it in this PR because my test needed to "root" a `JsBox`.